### PR TITLE
Feature/tao 8246/add form components/widget radiobox

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/TAO-8246

Requires: 
 - [x] https://github.com/oat-sa/tao-core-ui-fe/pull/5
 - [x] https://github.com/oat-sa/tao-core-ui-fe/pull/6

This PR is part of a bigger feature set, and is a split of https://github.com/oat-sa/tao-core/pull/2131.
The main purpose is to add a collection of form components.

So here's what we have here:
- `ui/form/widget/radioBox` is multi radio buttons widget

Unit tests:
```
npm i
npm run test form
```

Each unit test also comes with a visual playground so you can play with each component to see how they behave. To interact with them:
```
npm run test:keepAlive
```
Then open a browser on http://127.0.0.1:8082/test/form/widget/radioBox/test.html

### CODE REVIEW

File: src/form/widget/providers/radioBox.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    /**
+     * Gets the value of the widget
+     * @returns {String}
+     */
+    getValue() {
+        let value = this.getConfig().value || '';
+
+        if (this.is('rendered')) {
+            value = this.getElement()
+                .find(`.option input[name="${this.getUri()}"]:checked`)
```
Comment (krampstudio) [inline src/form/widget/providers/radioBox.js]: are you going to have multiple radios ? 
 `.option :checked` should be sufficient (like for the checkboxes) or is there particular reason to have the URI in the selector here ?

File: src/form/widget/providers/radioBox.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                .val() || '';
+        }
+
+        return value;
+    },
+
+    /**
+     * Sets the value of the widget
+     * @param {String} value
```
Comment (krampstudio) [inline src/form/widget/providers/radioBox.js]: do you expect the URI from the range ?

File: src/form/widget/providers/radioBox.js (hunk 1 above)
Comment (jsconan) [reply src/form/widget/providers/radioBox.js]: Good catch!

File: src/form/widget/providers/radioBox.js (hunk 2 above)
Comment (jsconan) [reply src/form/widget/providers/radioBox.js]: Reworded the JSDoc

**Pull request merged**